src/components/listing-page/product-card/ProductCard.js

```diff
@@ -1,33 +1,52 @@
 import './ProductCard.css';
 import { Link } from 'react-router-dom';
-import { useWishlist, useAuth, useAuthModal } from '../../../hooks';
+import { useWishlist, useAuth, useAuthModal, useCart } from '../../../hooks';
+import { priceFormatter } from '../../../utils';
 
 const ProductCard = props => {
   const { addToWishlist, removeFromWishlist, wishlist } = useWishlist();
   const { isAuth } = useAuth();
   const { showModal } = useAuthModal();
+  const { cart, addToCart } = useCart();
   const {
     title,
     brand,
     price,
-    quantity,
+    inStock,
     imageUrl,
     discount,
     exclusive,
     rating,
     _id: prodId,
   } = props.product;
 
+  const product = {
+    title,
+    brand,
+    price,
+    inStock,
+    imageUrl,
+    discount,
+    exclusive,
+    rating,
+    _id: prodId,
+  };
+
   const productCardClasses = exclusive
     ? 'card shadow ribbon ecom'
     : 'card shadow ecom';
 
-  const toggleWishListHandler = prodId => {
+  const toggleWishListHandler = () => {
     if (!isAuth) return showModal();
 
-    wishlist.items.includes(prodId)
+    wishlist.items.some(el => el._id === prodId)
```

Comment:
the naming can be improved here instead of adding el you can add wishlist.

Reply:
Thanks for pointing that out. I'll improve my naming convetion.